Management/CustomMusicManager.cs

```diff
@@ -81,20 +83,31 @@ public static void LoadSongsFromFolder(string parentFolderName, string folderNam
 			Dictionary<string, MusicItem> musicItemDict = new Dictionary<string, MusicItem>();
 
 			foreach (string text in from x in Directory.GetFiles(path)
-				where x.ToLower().EndsWith(".ogg") || x.ToLower().EndsWith(".wav") || x.ToLower().EndsWith(".mp3")
+				where x.ToLower().EndsWith(".ogg") || x.ToLower().EndsWith(".wav") || x.ToLower().EndsWith(".mp3") || x.ToLower().EndsWith(".txt")
 				select x)
 				{
 				string fileNameWithoutExtension = Path.GetFileNameWithoutExtension(text);
 
 				Plugin.LogInfo($"Found custom song: {parentFolderName}\\{folderName}\\{Path.GetFileName(text)}");
 
-				MusicItem music = new MusicItem
-				{
+				MusicItem music = new MusicItem {
 					id = "CUSTOM_" + fileNameWithoutExtension,
 					originalName = fileNameWithoutExtension,
 					resLocation = text,
 				};
 
+				// Files with a .txt extension will be redirected to the Music Bank folder to find music of the same name, with a naming convention of example.ogg.txt
+				if (Path.GetExtension(text) == ".txt") {
+					music.resLocation = Path.Combine(rootCustomSongsPath, Consts.musicBankFolderName, fileNameWithoutExtension).ToString();

```

Comment:
I don't think this would work because the resulting path doesn't include the file extension, but I think needs to here.

Reply:
The WithoutExtension only removes the .txt portion, the music format stays intact. Also if it does convert to string that's my bad, but I think it returns a type Directory.

Reply:
Oh right, that makes sense.
And yeah, Path.Combine() returns a String
<https://docs.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-5.0>